pkg/controller/modeladapter/modeladapter_controller.go

```diff
@@ -64,6 +65,8 @@ const (
 	ModelAdapterPodTemplateLabelKey   = "adapter.model.aibrix.ai/enabled"
 	ModelAdapterPodTemplateLabelValue = "true"
 
+	DefaultResyncInterval = 30 * time.Second
```

Comment:
@Jeffwan for now I have hard-coded resync interval; let me know if there is need to make it configurable.